tools/clang/test/CodeGenSPIRV/shader.debug.file.composite.hlsl

```diff
@@ -0,0 +1,30 @@
+// RUN: %dxc -E MainPs -T ps_6_0 -fspv-target-env=vulkan1.1 -fspv-debug=vulkan-with-source -spirv
+
+// Just check that compilation completes
```

Comment:
Would it be useful to add more detailed checks in order to prevent regressions, or will just checking that it compiles successfully suffice?

Reply:
Since the issue was that the compiler crashed (and hit an assert under Debug), I believe it is sufficient to just test that the compile completes.